### DISCUSSION TITLE
Need Help with ek9716 driver esp32

### DISCUSSION THREAD

**Discussion by snikesx** [Bodmer/TFT_eSPI#2424] [Compatible displays and setup files]
Need Help with ek9716 driver esp32
with GFX its work: 
Arduino_ESP32RGBPanel *bus = new Arduino_ESP32RGBPanel(GFX_NOT_DEFINED, GFX_NOT_DEFINED, GFX_NOT_DEFINED,41, 40, 39, 42, 14, 21, 47, 48, 45, 9, 46, 3, 8, 16, 1, 15, 7, 6, 5, 4);
Arduino_RPi_DPI_RGBPanel *gfx = new Arduino_RPi_DPI_RGBPanel(bus, 800, 0, 210, 30, 16, 480, 0, 22, 13, 10, 1, 16000000, true);

How i can use with TFT_eSPI?

**Comment by Bodmer** [comment]
The interface is desighed for an ESP32 S3 with a bespoke Espressif driver. That driver is not compatible with the TFT_eSPI library. There are no plans to add it as there are plenty of other display options and boards available.

**Comment by djhoke88** [comment]
@snikesx Hi, could you give me some information about the version of GFX that you used for test EK9716 driver ? 
if you could, it would help me to see some example code
Thanks
